app/models/submission.rb

```diff
@@ -30,12 +31,20 @@ class Submission < ApplicationRecord
         errors.add(:aasm_state, message: e.message)
       end
     end
+
+    event :replace_with_no_business do
+      transitions from: :completed, to: :replaced, guard: :not_no_business
+    end
   end
 
   def all_entries_valid?
     entries.validated.count == entries.count
   end
 
+  def not_no_business
```

Comment:
Seems to be lots of double negatives in here. Could this be renamed? Perhaps `replaceable?`